.github/workflows/fuzz.yml

```diff
@@ -0,0 +1,47 @@
+name: Fuzz
+on:
+  schedule:
+    # Once a week
+    - cron: '0 0 * * 0'
+  push:
+    paths:
+    - '.github/workflows/fuzz.yml'
+  # Allow manual trigger
+  workflow_dispatch:
+
+env:
+  CARGO_INCREMENTAL: 0
+  CARGO_NET_RETRY: 10
+  CI: 1
+  RUST_BACKTRACE: short
+  RUSTFLAGS: "-D warnings -W unreachable-pub -W bare-trait-objects"
+  RUSTUP_MAX_RETRIES: 10
+
+jobs:
+  rust:
+    if: github.repository == 'rust-lang/rust-analyzer'
+    name: Rust
+    runs-on: ubuntu-latest
+    env:
+      CC: deny_c
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+          fetch-depth: 20
+
+      - name: Install Rust toolchain
+        run: |
+          rustup update --no-self-update nightly
+          rustup component add rustfmt rust-src
+
+      - name: Build fuzzers
+        run: |
+          cargo install cargo-fuzz
+          cd crates/syntax
+          cargo +nightly fuzz build 
+          cd $GITHUB_WORKSPACE
+
+ 
```

Comment:
Nit: trailing whitespace or something.

Reply:
![image](https://user-images.githubusercontent.com/308347/214369219-b6dedbcf-3090-4587-9531-5daadf97caf1.png)

Looks the same, though.

Reply:
Apologies, that's weird. let me open it up in a different editor, I've just switched over to the helix editor, so I've likely just made a mistake somewhere in using it :)

Reply:
Yeah, Helix is weird at handling trailing newlines.

Reply:
Pretty sure that's sorted now. I opened up in vscode where I have trailing whitespace rendering turned on.